app/javascript/dashboard/components/widgets/conversation/filters/SortByFilter.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <woot-chat-list-filter
+    v-tooltip.left="'Sort conversations'"
+    :title="activeSortByLabel"
+    :dropdown-title="$t('CHAT_LIST.CHAT_SORT_BY_FILTER')"
+    :items="sortByItems"
+    :icon="activeSortBy ? 'arrow-sort' : ''"
+    :selected-value="activeSortBy"
+    @changeFilter="onTabChange"
+  />
+</template>
+
+<script>
+import wootConstants from 'dashboard/constants';
+import WootChatListFilter from './FilterItems';
+
+export default {
+  components: {
+    WootChatListFilter,
+  },
+
+  data: () => ({
+    activeSortBy: wootConstants.SORT_BY_TYPE.LATEST,
+    sortByItems: [
+      {
+        key: wootConstants.SORT_BY_TYPE.LATEST,
+        name: 'Last activity at',
+        label: 'Last activity',
+      },
+      {
+        key: wootConstants.SORT_BY_TYPE.CREATED_AT,
+        name: 'Created at',
+        label: 'Created at',
+      },
+      {
+        key: wootConstants.SORT_BY_TYPE.LAST_USER_MESSAGE_AT,
+        name: 'Last user message at',
+        label: 'Last message',
```

Comment:
Please add i18n here.

Reply:
@nithindavid I tried to add i18n getting some console errors. Thoughts?

Reply:
CC: @pranavrajs @scmmishra

Reply:
@iamsivin Can you please share the errors here?

Reply:
@pranavrajs 
<img width="902" alt="image" src="https://user-images.githubusercontent.com/64252451/211984015-c46f1610-5167-4c1e-b06e-a471de81ea60.png">

Reply:
I think this is because the data function is an arrow one @iamsivin

Reply:
@iamsivin Are you using `this.$t`?

Reply:
> @iamsivin Are you using `this.$t`?

Yes

Reply:
@iamsivin 

```
  data() {
    return {

    }
```
Will fix this

Reply:
Thanks working now.